cameraServer.py

```diff
@@ -36,8 +42,12 @@ def readCameraConfig(config):
 def readConfig():
     """Read configuration file."""
     # parse file
-    with open(configFile) as f:
-        j = json.load(f)
+
+    try:
+        with open(configFile) as f:
+            j = json.load(f)
+    except:
```

Comment:
You shouldn't  be using a bare except, because you want to know if it fails reading a file.